flake.nix

```diff
@@ -110,6 +110,15 @@
           type = "app";
           program = "${self.packages.${system}.default}/bin/llama-server";
         };
+        apps.llama-server-openai-proxy = {
+          type = "app";
+          program = "${
+            (let pythonWithPkgs = pkgs.python3.withPackages (ps: with ps; [ flask requests ]);
+             in pkgs.writeShellScriptBin "run-openai-proxy" ''
+                  ${pythonWithPkgs}/bin/python3 ${self}/examples/server/api_like_OAI.py
```

Comment:
This could've been a `makeWrapper`. I don't know if `api_like_OAI.py` takes any command line arguments, but `makeWrapper` would have passed those on